ui/src/app/pages/analysis/runtime-panel.spec.tsx

```diff
@@ -159,6 +159,55 @@ describe('RuntimePanel', () => {
     expect(runtimeApiStub.runtime.gceConfig.machineType).toEqual('n1-standard-4');
   });
 
+  it('should create runtime with preset values instead of getRuntime values if configurationType is GeneralAnalysis', async() => {
+    const gceConfig = defaultGceConfig();
+    gceConfig.machineType = 'n1-standard-16';
+    gceConfig.diskSize = 1000;
+    const runtime = {...runtimeApiStub.runtime,
+      status: RuntimeStatus.Deleted,
+      configurationType: RuntimeConfigurationType.GeneralAnalysis,
+      gceConfig: gceConfig,
+      dataprocConfig: null
+    };
+    runtimeApiStub.runtime = runtime;
+    act(() => { runtimeStore.set({runtime: runtime, workspaceNamespace: workspaceStubs[0].namespace}); });
+
+    const wrapper = await component();
+
+    await mustClickButton(wrapper, 'Create');
+
+    expect(runtimeApiStub.runtime.status).toEqual('Creating');
+    expect(runtimeApiStub.runtime.gceConfig.machineType).toEqual(runtimePresets.generalAnalysis.runtimeTemplate.gceConfig.machineType);
+    expect(runtimeApiStub.runtime.gceConfig.diskSize).toEqual(runtimePresets.generalAnalysis.runtimeTemplate.gceConfig.diskSize);
+  });
+
+  it('should create runtime with preset values instead of getRuntime values if configurationType is HailGenomicsAnalysis', async() => {
+    const dataprocConfig = defaultDataprocConfig();
+    dataprocConfig.masterMachineType = 'n1-standard-16';
+    dataprocConfig.masterDiskSize = 999;
+    dataprocConfig.workerDiskSize = 444;
+    dataprocConfig.numberOfWorkers = 5;
+
+    const runtime = {...runtimeApiStub.runtime,
+      status: RuntimeStatus.Deleted,
+      configurationType: RuntimeConfigurationType.HailGenomicAnalysis,
+      gceConfig: null,
+      dataprocConfig: dataprocConfig
+    };
+    runtimeApiStub.runtime = runtime;
+    act(() => { runtimeStore.set({runtime: runtime, workspaceNamespace: workspaceStubs[0].namespace}); });
+
+    const wrapper = await component();
+
+    await mustClickButton(wrapper, 'Create');
+
+    expect(runtimeApiStub.runtime.status).toEqual('Creating');
+    expect(runtimeApiStub.runtime.dataprocConfig.masterMachineType).toEqual(runtimePresets.hailAnalysis.runtimeTemplate.dataprocConfig.masterMachineType);
```

Comment:
opt: this is a bit verbose, consider using an approach like this to do a partial comparison: https://github.com/all-of-us/workbench/blob/a6e79859078446f570ce1da64ba905c06231f9cf/ui/src/app/pages/analysis/runtime-panel.tsx#L150-L154

There may be a better way, but I haven't seen any dedicated test utils for doing so.